src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
```

Comment:
Check in with @wdo3650 about this, turns out he wasn't building the Sass and it works fine if you do.